docs/pages/blog/danail-hadjiatanasov-joining.md

```diff
@@ -4,7 +4,7 @@ description: We are excited to share that Danail Hadjiatanasov has joined MUI as
 date: 2020-10-23T00:00:00.000Z
 authors: ['oliviertassinari']
 card: true
-tags: ['Team']
+tags: ['Company', 'MUI X', 'News']
```

Comment:
Implements the proposals from https://www.notion.so/mui-org/Improve-blog-infrastructure-bcae2b15493f4daba0758adf499f7793#c777a90341e0489fab4ba9414a60e995